mantle/system/ns/enter.go

```diff
@@ -60,7 +60,9 @@ func Create() (netns.NsHandle, error) {
 		return netns.None(), err
 	}
 	defer origns.Close()
-	defer netns.Set(origns)
+	defer func() {
+		err = netns.Set(origns)
```

Comment:
Prior assignment to `err` will be swallowed by the defer. `err` should be wrapped.

Reply:
Make sense. Updated.